guides/working-with-jooq-flyway-using-testcontainers/metadata.json

```diff
@@ -2,7 +2,7 @@
   "title": "Working with jOOQ and Flyway using Testcontainers",
   "intro": "This guide shows how to create a Micronaut application with jOOQ support",
   "authors": ["Sergio del Amo"],
-  "tags": ["spring-boot"],
+  "tags": ["flyway", "jooq","testcontainers"],
```

Comment:
tags are already added as tags. Why duplicated it here?

Reply:
should it just be an empty or omitted `tags` entry then?

Reply:
`"tags": []`